frontend/individual_pedigree.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file This file contains the Pedigree function. This function fetches
+ *       pedigree for a given `id` (parsed from the url).
+ */
+
+import React, { Component, useEffect } from 'react'
+import { Link } from "react-router-dom";
+import { get } from './communication';
+import { PedigreeNetwork } from "./pedigree"
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+
+
+/**
+ * Shows the information of a given individual and the pedigree graph built using the vis-network component
+ */
+export function IndividualPedigree({ id, generations }: { id: string }) {
+  const [pedigree, setPedigree] = React.useState(undefined as any)
+  const [individual, setIndividual] = React.useState(undefined as any)
+  const [generations_input, setGenerations] = React.useState(generations)
+
+
+  React.useEffect(() => {
+
+    let mounted = true
+    const url = `/api/individual/${id}`
+    get(url).then(
+      data => {
+        mounted && data && setIndividual(data)
+      },
+      error => console.error(error)
+    )
+    get(`/api/pedigree/${id}/${generations_input}`).then(
+      data => mounted && data && setPedigree(data),
+      error => console.error(error)
+    )
+
+    return () => { mounted = false }
+
+  }, [generations_input])
```

Comment:
I would add `id` to the depedency list since there is a dependency.

Reply:
ok